internal/security/proxy/config/config.go

```diff
@@ -129,10 +131,26 @@ func (c *ConfigurationStruct) UpdateWritableFromRaw(rawWritable interface{}) boo
 // structure -- until we can make backwards-breaking configuration.toml changes (which would consolidate these fields
 // into an interfaces.BootstrapConfiguration struct contained within ConfigurationStruct).
 func (c *ConfigurationStruct) GetBootstrap() interfaces.BootstrapConfiguration {
+	//To keep config file for proxy unchanged in Geneva we need to create a temporary SecretStore struct so that bootstrapHandler can use it to create a secret client
+	//The config file may be changed in the future version and SecretStore can be used directly like other core services
+	ss := bootstrapConfig.SecretStoreInfo{
+		Host:                    c.SecretService.Server,
+		Port:                    c.SecretService.Port,
+		Path:                    c.SecretService.CertPath,
+		Protocol:                "https",
+		RootCaCertPath:          c.SecretService.CACertPath,
```

Comment:
Suggest to having input validation on the required configuration items; eg. the path should not be empty and should be well-formed because now the Protocol is https for sure.

Reply:
Mark this resolved per Bryon's comments.